lib/config/presets/local/index.ts

```diff
@@ -12,6 +13,7 @@ const resolvers = {
   [PlatformId.Azure]: azure,
   [PlatformId.Bitbucket]: bitbucket,
   [PlatformId.BitbucketServer]: bitbucketServer,
+  [PlatformId.CodeCommit]: codecommit,
```

Comment:
i think we can remove the reexport file and use common here directly

Reply:
i dont know what that means, i just added it like the others care to elaborate on what should happen here?

Reply:
@viceice do you mean use `../codecommit/common` ?

Reply:
```suggestion
  [PlatformId.CodeCommit]: local,
```

something like that with the other changes above. so we don't need the re-export file